lib/user.js

```diff
@@ -140,8 +140,16 @@ export class UserClient extends Client {
                             console.log(error);
                         });
                 };
-                upload(images.body, urls[0].url);
-                upload(images.head, urls[1].url);
+                urls.forEach((url) => {
+                    switch(url.key) {
```

Comment:
You can create a map between the key and the actual image key. Then you can do 

```js
const imageKey = map[url.key];
images[imageKey];
```

Reply:
Would this have any performance benefits? The key currently doesn't match the images key so this would have to change also

Reply:
It would as this is just a map lookup, while the switch have a comparison per branch